apps/docs-snippets2/src/types/bits512/b512-in-the-sdk.ts

```diff
@@ -0,0 +1,12 @@
+// #region snippet-1
+import { Provider, Wallet } from 'fuels';
+
+import { LOCAL_NETWORK_URL } from '../../env';
+
+const provider = await Provider.create(LOCAL_NETWORK_URL);
+const wallet = Wallet.generate({ provider });
+
+console.log('public key', wallet.publicKey);
+
+// 0x97e3a666e4cd34b6b3cf778ef5ec617de4439b68f7a629245442a1fece7713094a1cb0aa7ad0ac253ca1ea47d4618f9090b2a881e829e091fb2c426763e94cca
```

Comment:
@petertonysmith94 This is being used as an example rather than the actual value.

Do you think it is better to use something like this?

```ts
// 0x...
```

Reply:
@Torres-ssf I believe the full B512 hex string makes sense for now.